test/cli_imports_test.py

```diff
@@ -102,18 +113,29 @@ def test_import_object(dir_structure, ref, expected_object_type, mock_dir):
         assert isinstance(_translated_obj, expected_object_type)
 
 
-def test_import_package_properly():
-    # https://modalbetatesters.slack.com/archives/C031Z7H15DG/p1664063245553579
-    # if importing pkg/mod.py, it should be imported as pkg.mod,
```

Comment:
The old comment behavior changes with this PR, as it wasn't in line with how `python pkg/mod.py` works.
If users want to keep getting the fully qualified python path of the imported module, they should use `modal run pkg.mod` instead.

Reply:
Sadly the betatesters link here doesn't work anymore, so I'm not sure about how this change afffects the mentioned use case